playground/android/gradlew.bat

```diff
@@ -13,15 +13,18 @@ if "%DIRNAME%" == "" set DIRNAME=.
 set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
+@rem Resolve any "." and ".." in APP_HOME to make it shorter.
+for %%i in ("%APP_HOME%") do set APP_HOME=%%~fi
+
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m" "-Xms64m"

```

Comment:
this file should not be pushed

Reply:
Same thing, it's based on RN 0.65.1's default app
https://github.com/react-native-community/rn-diff-purge/blob/release/0.65.1/RnDiffApp/android/gradlew